qa/tasks/cephadm.py

```diff
@@ -819,6 +822,31 @@ def ceph_bootstrap(ctx, config):
                     '*', '--mode', '0755'],
                    check_status=False)
 
+        # default logrotate config for cluster logs does not work for teuthology
+        # where debug logging tends to be enabled and massive log files that
+        # are multiple Gigabytes can be generate in tests that last an hour or two
+        teuth_cluster_logrotate_config = """# created by cephadm
+/var/log/ceph/{{ fsid }}/*.log {
+    rotate 100
+    size 10G
+    compress
+    sharedscripts
+    postrotate
+        killall -q -1 {{ targets|join(' ') }} || pkill -1 -x '{{ targets|join('|') }}' || true
+    endscript
+    missingok
+    notifempty
+    su root root
+}"""
+        bootstrap_remote.run(args=[
+            'echo', teuth_cluster_logrotate_config, run.Raw('>'), '/root/logrotate.conf'
+        ])
+        _shell(ctx, cluster_name, bootstrap_remote,
+               ['ceph', 'orch', 'write-custom-logrotate', 'cluster', '-i', '/mnt/logrotate.conf'],
```

Comment:
There is a divergence in configuration behavior with ceph.py in that this task is not using

https://github.com/ceph/ceph/blob/69bd270cf53ee6fe98ee2c0e86624bd85040149f/qa/tasks/logrotate.conf

I don't think, in practice, anyone bothers to configure that in any significant way so a static config is okay with me.